alembic/versions/9359297269a8_convert_ribbon_to_a_multichoice_column.py

```diff
@@ -0,0 +1,38 @@
+"""Convert ribbon to a multichoice column.
+
+Revision ID: 9359297269a8
+Revises: 167243c0e86c
+Create Date: 2017-07-21 09:18:38.444238
+
+"""
+
+
+# revision identifiers, used by Alembic.
+revision = '9359297269a8'
+down_revision = '167243c0e86c'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+
+try:
+    is_sqlite = op.get_context().dialect.name == 'sqlite'
+except:
+    is_sqlite = False
+
+if is_sqlite:
+    op.get_context().connection.execute('PRAGMA foreign_keys=ON;')
+    utcnow_server_default = "(datetime('now', 'utc'))"
+else:
+    utcnow_server_default = "timezone('utc', current_timestamp)"
+
+
+def upgrade():
+    op.alter_column('attendee', 'ribbon', type_=sa.Unicode(), server_default = '', nullable = False)
```

Comment:
Since we're removing c.NO_RIBBON altogether, this upgrade script will have to do a little data massaging. We need to set ribbon to an empty string if ribbon == "154973361".

I have no idea if the following will actually work, but the principles are sound. I copied this from this [blog post](http://www.georgevreilly.com/blog/2016/09/06/AlembicDataMigrations.html):

```python
from sqlalchemy.sql import table

def upgrade():
    op.alter_column('attendee', 'ribbon', type_=sa.Unicode(), server_default = '', nullable = False)

    attendee = table(
        'attendee',
        sa.Column('ribbon', sa.Unicode())
        # Other columns not needed for the data migration
    )

    op.execute(
        attendee
            .update()
            .where(attendee.c.ribbon == '154973361')
            .values({'ribbon': ''})
    )
```

In order to test the migration, I'd like to make sure the staging servers have at least one attendee with each type of ribbon (including c.NO_RIBBON). Then after deploying to staging, we need to verify that the attendees have maintained their expected ribbons (or lack thereof).